exasol_transformers_extension/resources/templates/with_spans/span_token_classification_udf.jinja.sql

```diff
@@ -0,0 +1,33 @@
+CREATE OR REPLACE {{ language_alias }} SET SCRIPT "TE_TOKEN_CLASSIFICATION_UDF_WITH_SPAN"(
+    device_id INTEGER,
+    bucketfs_conn VARCHAR(2000000),
+    sub_dir VARCHAR(2000000),
+    model_name VARCHAR(2000000),
+    text_data VARCHAR(2000000),
+    docid INTEGER,
```

Comment:
```suggestion
    text_data_docid INTEGER,
```

All columns of one span should have the same prefix, such that we can later introduce something like a span column abstraction when we describe tables with spans. This also means that the suffix for the three columns is also defined. Your choice was the correct one, docid, char_begin, char_end